galsim/gsobject.py

```diff
@@ -1225,8 +1225,8 @@ def _get_new_bounds(self, image, nx, ny, bounds, center):
         elif nx is not None and ny is not None:
             b = BoundsI(1,nx,1,ny)
             if center is not None:
-                b = b.shift(_PositionI(np.floor(center.x+0.5)-b.center.x,
-                                       np.floor(center.y+0.5)-b.center.y))
+                b = b.shift(_PositionI(np.floor(center.x+0.5-b.true_center.x),
+                                       np.floor(center.y+0.5-b.true_center.y)))
```

Comment:
Worth mentioning that [here](https://github.com/GalSim-developers/GalSim/blob/releases/2.6/galsim/gsobject.py#L1167) is the other line, which this should be consistent with (and wasn't).